tests/chainerx_tests/unit_tests/routines_tests/test_connection.py

```diff
@@ -31,6 +31,19 @@ def _create_conv_args(
     return x, w, b, stride, pad, cover_all
 
 
+def _convert_data_format(array, data_format):
```

Comment:
Please write explanation. In particular, what's the assumption about `array`? Should it be contiguous? I don't think it is always contiguous.

Reply:
I'll rename the argument and add assertions and comments.